### PR TITLE
lua integration

### LINKED ISSUE #13: Integrate lua interpreter

### PR DESCRIPTION
**Pull request by BaderBC**
Closes #13
- lua integrated
- api.Package fixed

### CODE REVIEW

File: src/lua/api_binding.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package lua
+
+import (
+	"github.com/nasz-elektryk/spito-rules/api"
+	"github.com/yuin/gopher-lua"
+	"reflect"
+)
+
+// Every api needs to be attached here in oder to be available:
```
Comment (maxidragon) [inline src/lua/api_binding.go:9]: there is a typo 😉
```suggestion
// Every api needs to be attached here in order to be available:
```

File: src/lua/interpreter.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+const devMode = false
+
+func DoesRulePasses(script string) (bool, error) {
+	L := lua.NewState(lua.Options{SkipOpenLibs: !devMode})
+	defer L.Close()
+	attachApi(L)
+
+	if err := L.DoString(script); err != nil {
+		// TODO: think about better error handling!
+		panic(err)
```
Comment (cooligus) [inline src/lua/interpreter.go:18]:
```go
return false, err
```

**Review by cooligus — CHANGES_REQUESTED** [review]
Don't forget to add GetDaemon

**Review by cooligus — APPROVED** [review]

**Review by maxidragon — APPROVED** [review]
Please wait for @cooligus approval before merging

**Pull request merged**